pkg/cmd/server.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The MayaData Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+	"strings"
+
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	// defaultOpenebsNS default openebs namespace
+	defaultOpenebsNS string = "openebs"
+
+	// defaultVeleroNs default velero namespace
+	defaultVeleroNs string = "velero"
+
+	// defaultClientQPS for dmaas-operator
+	defaultClientQPS float32 = 20.0
+
+	// defaultClientBurst for dmaas-operator
+	defaultClientBurst int = 30
+)
+
+type serverOpts struct {
+	openebsNamespace string
+	veleroNamespace  string
+	clientQPS        float32
+	clientBurst      int
+	logLevel         string
+}
+
+func newServerOpts() *serverOpts {
+	return &serverOpts{
+		openebsNamespace: defaultOpenebsNS,
+		veleroNamespace:  defaultVeleroNs,
+		clientBurst:      defaultClientBurst,
+		clientQPS:        defaultClientQPS,
+		logLevel:         logrus.InfoLevel.String(),
+	}
+}
+
+func (s *serverOpts) BindFlags(flags *pflag.FlagSet) {
+	flags.StringVar(&s.openebsNamespace, "openebs-namespace",
+		s.openebsNamespace,
```

Comment:
use environment variable OPENEBS_NAMESPACE, VELERO_NAMESPACE if not given through cmd option